flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -102,12 +104,21 @@
 			.withDescription("The maximum number of prior execution attempts kept in history.");
 
 	/**
-	 * The maximum number of prior execution attempts kept in history.
+	 * This options specifies the failover strategy.
 	 */
 	public static final ConfigOption<String> EXECUTION_FAILOVER_STRATEGY =
 		key("jobmanager.execution.failover-strategy")
 			.defaultValue("full")
-			.withDescription("The maximum number of prior execution attempts kept in history.");
+			.withDescription(Description.builder()
+				.text("This options specifies the job computation recovers from task failures.")
+				.linebreak()
+				.text("Accepted values are:")
+				.linebreak()
```

Comment:
I think lists always render in a separate line. So I would say this `linebreak` is unnecessary. The linebreak before `Accepted values are:` is also questionable right now. It renders like this:

![screenshot from 2018-08-07 15-46-16](https://user-images.githubusercontent.com/6242259/43779768-6caea624-9a59-11e8-8d50-6e607d3825e4.png)

I would either remove the line break or add two linebreaks to render empty line.